extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -138,7 +162,9 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
             updates.push({ name } as ContactUpdate);
           } else {
             // No public key found. Returning early, nothing to update in local store below.
-            return existingContact ?? await ContactStore.obj({ email });
+            if (existingPubKey) return existingPubKey;
+            const contact = await ContactStore.obj({ email });
+            return ContactUtil.toPubKeyInfo(contact);
```

Comment:
These to lines will simply return `undefined`. Thus `toPubkeyInfo()` is not required

Reply:
Why these two lines simply return `undefined`?? Sorry, but I can't understand this at the moment. Can you please provide more detailed explanation explanation? (p.s. I just was trying to port existing code without losing any already implemented functionality.)